### ISSUE TITLE
Unsupported parameters for (ansible.legacy.command) module: warn.

### ISSUE THREAD

**Issue by lz-wang** [metno/ansible-role-chrony#19]
Hi!

I've read `CODE_OF_CONDUCT.md` in this repository, and want to report a [bug/issue].

[Description]
ansible [core 2.16.3]

when running this role, an error happend, the msg is

```
TASK [chrony : Check ntpd is running] ******************************************
fatal: [centos_7]: FAILED! => {
   changed: false,
   changed_when_result: "The conditional check 'ntpd_running.rc == 0' failed. The error was: error while evaluating conditional (ntpd_running.rc == 0): 'dict object' has no attribute 'rc'. 'dict object' has no attribute 'rc'",
   msg: "Unsupported parameters for (ansible.legacy.command) module: warn. Supported parameters include: _raw_params, _uses_shell, argv, chdir, creates, executable, removes, stdin, stdin_add_newline, strip_empty_ends.",
}
```

After I comment the following lines in `tasks/main.yml`, this error can be bypassed
```yaml
- name: Check ntpd is running
  ansible.builtin.shell: |
    pgrep ntpd
  # args:
  #   warn: false
  register: ntpd_running
  changed_when: ntpd_running.rc == 0
  failed_when: false
  when: chrony_disable_ntpd and ansible_distribution|lower != "fedora"
```

Could you please fix this issue?

Thanks for your very helpful ansible-role-chrony!

**Comment by KjetilKnutsen** [comment]
Hi
Thanks for the feedback regarding the deprecation of `warn`. Our roles are currently only officially supporting ansible up to version 2.12, but most should work for 2.16 aswell. 

We are currently working on upgrading the versions to officially support 2.16 aswell, and will at that time fix this issue.

**Comment by KjetilKnutsen** [comment]
This has now been fixed in version 4.0.0. Closing issue.
